ui/src/components/DownloadButton/OpenApiDownloadBtn.tsx

```diff
@@ -1,17 +1,13 @@
 import React from 'react';
-import { createRESTURL } from '@splunk/splunk-utils/url';
-import { app } from '@splunk/splunk-utils/config';
 import Icon from '@splunk/react-icons/ArrowBroadUnderbarDown';
+import { getBuildDirPath } from '../../util/script';
 import DownloadButton from './DownloadButton';
 
 function OpenApiDownloadButton() {
     return (
         <div>
             <DownloadButton
-                fileUrl={createRESTURL('static/openapi.json', {
-                    app,
-                    owner: 'nobody',
-                })}
+                fileUrl={getBuildDirPath().replace('js/build', 'openapi.json')}
```

Comment:
@artemrys  here is the way how it is different, generally it should work as it works for globalConfig.json and from what i can see it has been like that since 3 years. Just not so elegant way to find this endpoint but after briefly looking there isn't any better way to find the package id.
Also wasn't able to identify any more items in UI part connected to openapi, so unless we want to change something that should be it